src/notifications/generate.ts

```diff
@@ -94,6 +94,9 @@ export const notificationTitleMap: Record<
   collection_updated: (ctx: NotificationPostContext) =>
     `The collection "<b>${ctx.post.title}</b>" just got updated with new details`,
   dev_card_unlocked: () => 'DevCard unlocked!',
+  source_post_added: (
```

Comment:
This type of message might be weird UX if source adds more then 1 article in one go.
You would get multiple entries that all are the same (I know different post, but used doesn't know this)

Reply:
Agree, we can easily show title/attachment of the post. Would that make more sense? If yes I can ask product.

Reply:
Maybe, if you can double check if that's an option?

Maybe propose something like this?

![Screenshot 2024-03-07 at 09 55 41](https://github.com/dailydotdev/daily-api/assets/554874/075a653a-bae3-44ce-8504-5378319324fe)

Reply:
Yes, just like squads, will raise 👌

Reply:
Updated in 6e664cf3c9a46537d6c1fcc4e158073aa8cff9d4, added updated notification look in PR description.